docs/proposal/tls_cert.md

```diff
@@ -0,0 +1,140 @@
+## Certificate Application and Management Module
+
+### Summary
+
+kmesh supports TLS capabilities that require the use of certificates issued by istiod, so a set of certificate application and management modules is needed to apply for certificates from istiod and manage the lifecycle of the certificates.
```

Comment:
How do you translate these sensences,  gpt? Look not that native

Reply:
yes, and I checked again